templates/sample_stubs/meta-aws.yml

```diff
@@ -0,0 +1,44 @@
+---
+meta:
+  deployment_name: REPLACE_WITH_CF_DEPLOYMENT_NAME
+  director_uuid: REPLACE_WITH_DIRECTOR_ID
+  release_name: cf-redis
+  apps_domain: REPLACE_WITH_APP_DOMAIN
+  external_domain: REPLACE_WITH_DOMAIN
+  syslog_aggregator:
+    address: REPLACE_WITH_SYSLOG_AGGREGATOR_HOST
+    port: REPLACE_WITH_SYSLOG_AGGREGATOR_PORT
+  broker:
+    name: redis
+    username: REPLACE_WITH_BROKER_USERNAME
+    password: REPLACE_WITH_BROKER_PASSWORD
+    host: (( "redis-broker" "." .meta.external_domain ))
+  cf:
+    admin_username: REPLACE_WITH_ADMIN_USERNAME
+    admin_password: REPLACE_WITH_ADMIN_PASSWORD
+    api_url: (( "https://api" "." .meta.external_domain ))
+    apps_domain: (( .meta.apps_domain ))
+    skip_ssl_validation: false # Optional, change to `true` in envs with self-signed SSL certs
+    nats:
```

Comment:
The `nats` section now lives one level higher, at the same level as `cf`. Simply indenting this block one level to the left would meet the new requirements for deploying the `cf-redis-release`